essentials/configurations.mdx

```diff
@@ -36,6 +38,7 @@ Here is an example of what you can add to your `squire.config.json` file:
 		"Please do not ask me to \"make sure\" of anything",
 		"Make sure all database options use custom repositories"
 	],
+    "introComment": false,
```

Comment:
We should set this to true, we want people to get the intro comment if they copy our config.

Reply:
Done